core/Command/Background/ListCommand.php

```diff
@@ -67,6 +67,7 @@ protected function configure(): void {
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
 		$jobs = $this->jobList->getJobsIterator($input->getOption('class'), (int)$input->getOption('limit'), (int)$input->getOption('offset'));
+		$output->writeln("Output is currently limited to first " .  $input->getOption('limit') . " jobs. Specify `-l, --limit[=LIMIT]` to override.");
```

Comment:
```suggestion
		if ($input->getOption('output') === self::OUTPUT_FORMAT_PLAIN) {
			$output->writeln("Output is currently limited to first " .  $input->getOption('limit') . " jobs. Specify `-l, --limit[=LIMIT]` to override.");
		}
```

To output the warning only for plain.

Reply:
Looks like a good solution for now. Let's get this in, no?